lib/api_message.go

```diff
@@ -0,0 +1,52 @@
+package sebak
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"boscoin.io/sebak/lib/network"
+)
+
+func (api NetworkHandlerNode) MessageHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
The `MessageHandler`, `BallotHandler`, `NodeInfoHandler` and `ConnectHandler` have same prifix(UrlPathPrefixNode).
So I think defining these four handlers in lib/api_node.go is better for readability.